### DISCUSSION TITLE
Projects (beta) - Hot reload

### DISCUSSION THREAD

**Discussion by lc-caigwatkin** [community/community#5738] [Projects and Issues] [answered]
Often it is useful to collaborate when planning a project. It would be very useful to have the views hot reload instead of needing to refresh the page.

**Comment by HenrikOssipoff** [comment]
This was actually one of the first things we noticed after trying out the beta of Projects. While more of a quality of life change to our organisation, it's something we've come to know from other software. Would definitely appreciate automatic real-time updates across projects.

**Comment by ynouri** [comment]
Our engineering team is migrating from Notion to Projects Beta and we have many multiple collaborative planning sessions to refine a Sprint content. In my opinion, the hot reload / live update feature is a must-have as it makes collaboration 10x easier. It's such an important feature to us that we are considering using only Github Issues at the database / source of truth for tickets, and write a Notion integration to benefit from the visualization / collaborative features like timeline or hot reload

**Comment by azenMatt** [comment]
We are working on this - give us a little more time and it will be with you soon :)

**Comment by azenMatt** [comment, marked as answer]
I missed updating this one when we launched this last month! But hopefully you have all noticed that all the views now hot reload 😄
